app/assets/javascripts/kassi.js

```diff
@@ -1065,7 +1068,7 @@ function set_textarea_maxlength() {
 
       }
     })
-    .live('keydown', function(event) {
+    .on('keydown', function(event) {
```

Comment:
Yep, `live` was exactly for that purpose. In this case the event handler is attached directly to the textarea element and thus there will be no dynamically added children there, so it's safe to use `on`.